be/src/exprs/runtime_filter.h

```diff
@@ -95,10 +95,13 @@ class SimdBlockFilter {
     // The filter is divided up into Buckets:
     static constexpr int BITS_SET_PER_BLOCK = 8;
     using Bucket = uint32_t[BITS_SET_PER_BLOCK];
+    static constexpr size_t MINIMUM_ELEMENT_NUM = 1UL;
 
     SimdBlockFilter() = default;
 
-    ~SimdBlockFilter() noexcept { free(_directory); }
+    ~SimdBlockFilter() noexcept {
+        if (_directory) free(_directory);
```

Comment:
_directory = nullptr ?

Reply:
> _directory = nullptr ?

if we invoke clear method, _directory will set to nullptr, here we should avoid double free.

```
void SimdBlockFilter::clear() {
    if (_directory) {
        free(_directory);
        _directory = nullptr;
        _log_num_buckets = 0;
        _directory_mask = 0;
    }
}
```